### PR TITLE
Bug fixing

### PR DESCRIPTION
**Pull request by sladomic**

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -52,7 +52,8 @@
             android:name=".content.task.StarActivity"
             android:screenOrientation="fullSensor" />
         <activity android:name=".content.task.LessonCompleteActivity" />
-        <activity android:name=".authentication.StudentImageCollectionActivity" />
+        <activity android:name=".authentication.StudentImageCollectionActivity"
+            android:screenOrientation="landscape"/>
```
Comment (jo-elimu) [inline app/src/main/AndroidManifest.xml:56]: Does this _require_ landscape mode during authentication?

**Review by jo-elimu — APPROVED** [review]

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (sladomic) [reply app/src/main/AndroidManifest.xml:56]: @jogrimst  at the moment yes. If we want to change it we would also need to change the animalTemplate implementation (not a huge task but it's a change)

Comment (jo-elimu) [reply app/src/main/AndroidManifest.xml:56]: What happens if the tablet is held in portrait orientation while the authentication activity starts?

Comment (sladomic) [reply app/src/main/AndroidManifest.xml:56]: @jogrimst  with the change in the manifest it should switch to landscape  mode (like with some other apps on the tablet)

**Pull request merged**